src/compiler/helpers.js

```diff
@@ -149,3 +181,18 @@ export function getAndRemoveAttr (
   }
   return val
 }
+
+function setRange (
```

Comment:
A bit nit-picky, but `setRange` implies side-effect, `rangeSetItem` might be a better naming.